addons/event_crm/models/event_registration.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    lead_id = fields.Many2one('crm.lead', string='Lead', readonly=True)
+
+    multi = fields.Boolean(default=False)
```

Comment:
When we install event & crm, event_crm is installed automatically. So the creation of a lead is done in the create method for a registration. But when we install other module like event_sale, website_event, it's possible to add multiple registrations, but it creates always registration one by one. So to avoid the creation in the create method I need this field since the method to create_lead is called somewhere else.